superset/db_engine_specs/presto.py

```diff
@@ -561,10 +561,18 @@ def latest_partition(
             )
 
         column_names = indexes[0]["column_names"]
-        part_fields = [(column_name, True) for column_name in column_names]
-        sql = cls._partition_query(table_name, database, 1, part_fields)
-        df = database.get_df(sql, schema)
-        return column_names, cls._latest_partition_from_df(df)
+
+        return column_names, cls._latest_partition_from_df(
```

Comment:
Same logic as before just a cleaner presentation including adding keyword arguments, i.e., previously it wasn't apparent in `cls._partition_query(table_name, database, 1, part_fields)` what `1` and `part_fields` meant.